packages/terra-form-radio/src/RadioField.jsx

```diff
@@ -114,8 +114,14 @@ const RadioField = (props) => {
   const errorAriaDescriptionId = error ? `terra-radio-field-description-error-${uniqueid()}` : '';
   const ariaDescriptionIds = `${legendAriaDescriptionId} ${errorAriaDescriptionId} ${helpAriaDescriptionId}`;
 
+  let isSafariOrEdgBrowser = false;
+  if (navigator.userAgent.indexOf('Safari') !== -1 && (navigator.userAgent.indexOf('Chrome') === -1 || navigator.userAgent.indexOf('Edg') !== -1)) {
+    isSafariOrEdgBrowser = true;
+  }
```

Comment:
```suggestion
  const isSafariOREdge = ((navigator.userAgent.indexOf('Safari') !== -1 || navigator.userAgent.indexOf('Edg') !== -1) && navigator.userAgent.indexOf('Chrome') === -1);
```

Reply:
This condtion working fine in chrome and safari,but inEdge browser ( navigator.userAgent.indexOf('Chrome') this value is 81. navigator.userAgent.indexOf('Chrome') === -1); this condition always false in Edge browser.